src/main/resources/conf/chains/pottery-chain.conf

```diff
@@ -0,0 +1,160 @@
+{
```

Comment:
Is there stuff you can use from an existing chain conf perhaps? Then you could just include that and keep only the changed params.

Reply:
good idea...